assembly/src/procedures/mod.rs

```diff
@@ -64,18 +64,22 @@ impl Procedure {
     }
 
     /// Returns the number of memory locals reserved by the procedure.
-    #[allow(dead_code)]
     pub fn num_locals(&self) -> u32 {
         self.num_locals
     }
 
-    /// Returns a root of this procedure's MAST.
-    pub fn code_root(&self) -> &CodeBlock {
+    /// Returns the root of this procedure's MAST.
+    pub fn mast_root(&self) -> RpoDigest {
+        self.code_root.hash()
+    }
+
+    /// Returns a reference to the MAST of this procedure.
+    pub fn code(&self) -> &CodeBlock {
         &self.code_root
```

Comment:
Should we consider renaming the `code_root` field to `code` such that we have parity with the accessor?

Reply:
Agreed! I've actually fixed this in one of the subsequent PRs.